agenta-web/src/components/AlertPopup/AlertPopup.tsx

```diff
@@ -32,7 +33,9 @@ export default function AlertPopup({
     type,
     ...ModalProps
 }: AlertPopupProps) {
-    return Modal[type || "confirm"]({
+    const {modalInstance} = getAppValues()
+
+    return modalInstance[type || "confirm"]({
```

Comment:
can we please add a fallback here, that if modalInstance is nullish, we default to `Modal[type || "confirm"]`?

Reply:
Yep. will add fallback for modalInstance